include/RevRegFile.h

```diff
@@ -245,8 +256,10 @@ class RevRegFile {
   void SetPC(T val) {
     if( IsRV32 ){
       RV32_PC = static_cast<uint32_t>(val);
+      TRACE_PC_WRITE(RV32_PC);
     }else{
       RV64_PC = static_cast<uint64_t>(val);
+      TRACE_PC_WRITE(RV64_PC);
```

Comment:
See above. Here it's split between the destination register types. I think I favor this approach.